pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSTrain.java

```diff
@@ -95,15 +99,15 @@ public void trainModels(String trainingData) {
 
         // Run the learner
         for (int i = 0; i < iter; i++) {
-            System.out.println("Training round " + i);
+            logger.info("Training round " + i);
```

Comment:
STDOUT instead of logger -- not a huge amount of output, and probably run infrequently in a setting where the user wants to know what's going on.